<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ExponentialDialog</class>
    <widget class="QDialog" name="ExponentialDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>334</width>
                <height>199</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>140</x>
                    <y>140</y>
                    <width>168</width>
                    <height>30</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <widget class="QPushButton" name="cancelButton">
                        <property name="text">
                            <string>Cancel</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="okButton">
                        <property name="text">
                            <string>Ok</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>70</y>
                    <width>251</width>
                    <height>31</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                    <widget class="QLabel" name="label">
                        <property name="text">
                            <string>Your Lambda:</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QLineEdit" name="lambdaBox"/>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
